Data_publishing_requirements_in_science-Bronowicz

carobro opened the pull request:
**Handout on "Data publishing requirements in science".**

Includes topics like: DOI's, Possible types of publishing, 
Principles for publishing research data, ....

carobro committed:
Add files via upload

carobro committed:
Added Sources

nuest commented:
Danke für deine Mühen bis hierher, nur kleine Kommentare die du aber gerne erst im Nachgang der Vorstellung des Handouts berücksichtigen kannst:

- "Datenrepositorys": Plural entweder "Datenrepositorien" oder in denglisch "Datenrepositories"
- Verweise: Kannst du die Quellen auch bestimmten Abschnitten zuordnen? So muss ich als Leser erstmal herausfinden, welcher Link mir bei einer detaillierteren Frage weiter helfen könnte.
- Verweise sind teilweise seltsam formatiert, starten mit Jahreszahl in Klammern, und sind unvollständig. Zumindest Titel und Link sollte es immer geben.

nuest commented:
Bitte zwei Dinge nachholen:

1. Alle Autoren mit einem `@name` - Link im Handout erwähne, siehe Beispiel: https://github.com/Geosoft2/geosoft2-2018/blob/b8609b1ffb08f3e26be1809dfd6f9374c60d3663/agile-software-development/handout.md
1. Falls zwei Autoren, beide GitHub accounts im issue erwähnen via [@-mention](https://blog.github.com/2011-03-23-mention-somebody-they-re-notified/), damit sie Benachrichtigungen erhalten.

carobro committed:
@name ergänzt

carobro committed:
Update Data_publishing_requirements_in_science-Bronowicz.markdown

carobro committed:
Update Data_publishing_requirements_in_science-Bronowicz.markdown

carobro committed:
Update Data_publishing_requirements_in_science-Bronowicz.markdown

Quellenverzeichnis aktualisiert. 
Verweise fehlen noch

carobro committed:
Update

nuest commented:
Interessant für DOI + Repositorien: eine Übersicht über die Anzahl an DOIs die von verschiedenen Repositorien beantragt wurden: https://stats.datacite.org/#tab-datacentres

![image](https://user-images.githubusercontent.com/1325054/46736552-9f54d680-cc99-11e8-9b60-2ad3e8c0668f.png)

carobro committed:
Quellen den Abschnitten zugeordnet

carobro committed:
Link wurde hinzugefügt

https://stats.datacite.org/#tab-datacentres
Die Übersicht ist sehr interessant. Ich habe diesen mit in mein Handout übernommen
